### DISCUSSION TITLE
GSOC New Idea Proposal

### DISCUSSION THREAD

**Discussion by yagyesh-bobde** [camicroscope/caMicroscope#665] [Ideas]
I would like to propose a research topic. 
<h2>
Enhancing Pupillometry Through Advanced Video Processing
</h2>
The goal here would be to use advanced video processing to improve cognitive assessment.
<h3>
Background
</h3>
Traditional methods have limitations in capturing subtle changes in pupil dynamics, with video processing we can have a high frequency analysis of pupil dynamics.
<h3>
End Product
</h3>. 
An application for Pupillometry analysis and also combine it with other domains like neuroscience, psychology, and clinical diagnostics.

**Comment by birm** [comment]
This is an interesting project, but I'm not sure how well it fits into caMicroscope as it currently exists. I happen to know that some of the faculty at Emory Biomedical Informatics do research into what they call MCI (mild cognitive impairment), so that may be something you could work with them on.
Alternatively, if you see a good reason why this would be part of caMicroscope, which is focused on biomedical static image analysis in a spatial sense, please do share your thoughts!

**Reply by yagyesh-bobde** [reply]
Yes @birm, I asked you the slack channel. Actually I created a discussion there as well but got no response. I'm not sure how I should proceed. It would nice if you help and communicate.

**Reply by antriksh-9** [reply]
Yes @birm I also request you to that we should stick with slack for communicating.
[Slack](https://camicroscope.slack.com/archives/D068S5FTUAV/p1705332357321219)

**Comment by antriksh-9** [comment]
Hey @birm I am looking forward for participating in upcoming GSoC, so can you please tell me about the 2024 project idea lists?

**Reply by birm** [reply]
The 2024 Ideas list is sitll a draft, but it's up at https://github.com/camicroscope/GSOC/tree/2024
